third_party/blink/renderer/modules/mediasource/source_buffer_write_head.idl

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Non standard interface used for Cobalt only.
+// Returns the highest presentation timestamp written to the SbPlayer;
+// raises `InvalidStateError` if the SourceBuffer object has been removed
+// from the MediaSource object.
+
+[
+    ImplementedAs=SourceBufferWriteHead
```

Comment:
I am not sure the best convention for file names and class name here, i.e. shall we use SourceBufferWriteHead or CobaltSourceBuffer (and source_buffer_write_head.idl or cobalt_source_buffer.idl).

Just to bring this up, and please feel free to keep this as is for now or bring this to an internal sync for discussion.